File: python/logic2/round_sum.py
#codingbat link: https://codingbat.com/prob/p179960

def round_sum(a, b, c):
    return round10(a) + round10(b) + round10(c)
  
  
def round10(num):
    digit = 0
    newNum = num
    if(newNum // 10 == 0): 
        digit = newNum

    while(newNum // 10 != 0):
        if(newNum // 10 == 0):
            break
        else:
            digit = newNum % 10
            newNum = newNum // 10
    
    if(digit >= 5):
        return num + (10 - digit)
    else:
        return num - digit

print(round_sum(6, 4, 4))

File: java_code/logic2/roundSum.java
package java_code.logic2;

//codingbat link: https://codingbat.com/prob/p186753

public class roundSum{

    public static int roundSum(int a, int b, int c){
        return round10(a) + round10(b) + round10(c);
    }

    public static int round10(int num){
        int number = num;

        while(number > 10){
            number = number%10;
        }

        if(number >= 5){
            return num + (10 - number);
        } else {
            return num - number;
        }

    }

    public static void main(String[] args){
        System.out.println(roundSum(16, 17, 18)); //expected output: 60
        System.out.println(roundSum(12, 13, 14)); //expected output: 30
        System.out.println(roundSum(6, 4, 4)); //expected output: 10
    }

}